components/inputs/test/input-select.visual-diff.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../typography/typography.js';
+		</script>
+		<title>d2l-input-select visual-diff tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<style>
+			select.vui-input {
+    border-radius: .3rem;
+    border-style: solid;
+    box-sizing: border-box;
+    display: inline-block;
+    height: auto;
+    margin: 0;
+    vertical-align: middle;
+    transition: background-color .5s ease,border-color 1ms ease;
```

Comment:
Watch out for possible erroneous diffs due to this.  It looks like there isn't necessarily always a change though.

Reply:
Right, good reminder -- I'm removing this transition in the new code as well, since the background-color doesn't actually change and a `1ms` border colour change is imperceivable.